docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -0,0 +1,366 @@
+### Cohere Embedding Connector Blueprint:
+
+This blueprint will show you how to connect a Cohere embedding model to your Opensearch instance, including creating a k-nn index and your own Embedding pipeline. You will require a Cohere API key.
+
+Cohere currently offers the following Embedding models (with model name and embedding dimensions). Note that only a couple are tested.
```

Comment:
> Note that only a couple are tested.

Do you have any plan for the untested models?

Reply:
@ylwu-amzn Hey sorry, just had some time to check this out again. I removed the other untested models and will make a note to test out the other ones when time clears up, I've addressed all the other PR comments

Reply:
@ylwu-amzn is it alright to merge this PR standalone, and update the doc again later when I get a chance to test?